stubs/fpdf2/fpdf/fpdf.pyi

```diff
@@ -185,37 +181,37 @@ class FPDF:
         format: _Format | tuple[float, float] = ...,
         same: bool = ...,
         duration: int = ...,
-        transition: Any | None = ...,
+        transition: Incomplete | None = ...,
     ) -> None: ...
     def header(self) -> None: ...
     def footer(self) -> None: ...
     def page_no(self) -> int: ...
-    def set_draw_color(self, r, g: int = ..., b: int = ...) -> None: ...
-    def set_fill_color(self, r, g: int = ..., b: int = ...) -> None: ...
-    def set_text_color(self, r, g: int = ..., b: int = ...) -> None: ...
-    def get_string_width(self, s, normalized: bool = ..., markdown: bool = ...): ...
+    def set_draw_color(self, r: int, g: int = ..., b: int = ...) -> None: ...
+    def set_fill_color(self, r: int, g: int = ..., b: int = ...) -> None: ...
+    def set_text_color(self, r: int, g: int = ..., b: int = ...) -> None: ...
+    def get_string_width(self, s: str, normalized: bool = ..., markdown: bool = ...) -> float: ...
     def set_line_width(self, width: float) -> None: ...
     def line(self, x1: float, y1: float, x2: float, y2: float) -> None: ...
-    def polyline(self, point_list, fill: bool = ..., polygon: bool = ...) -> None: ...
-    def polygon(self, point_list, fill: bool = ...) -> None: ...
+    def polyline(self, point_list: list[tuple[float, float]], fill: bool = ..., polygon: bool = ...) -> None: ...
+    def polygon(self, point_list: list[tuple[float, float]], fill: bool = ...) -> None: ...
     def dashed_line(self, x1, y1, x2, y2, dash_length: int = ..., space_length: int = ...) -> None: ...
-    def rect(self, x, y, w, h, style: Any | None = ...) -> None: ...
-    def ellipse(self, x, y, w, h, style: Any | None = ...) -> None: ...
-    def circle(self, x, y, r, style: Any | None = ...) -> None: ...
+    def rect(self, x: float, y: float, w: float, h: float, style: Incomplete | None = ...) -> None: ...
+    def ellipse(self, x: float, y: float, w: float, h: float, style: Incomplete | None = ...) -> None: ...
+    def circle(self, x: float, y: float, r, style: Incomplete | None = ...) -> None: ...
     def add_font(self, family: str, style: _FontStyle = ..., fname: str | None = ..., uni: bool = ...) -> None: ...
     def set_font(self, family: str | None = ..., style: _FontStyles = ..., size: int = ...) -> None: ...
     def set_font_size(self, size: int) -> None: ...
-    def set_stretching(self, stretching) -> None: ...
-    def add_link(self): ...
+    def set_stretching(self, stretching: float) -> None: ...
```

Comment:
`font_stretching` on line 92 of this stub needs to be changed to be `float` rather than `int`: https://github.com/PyFPDF/fpdf2/blob/48f837963b7ae1956a0e5b23815063d23c1964f5/fpdf/fpdf.py#L1922